spring-cloud-kubernetes-integration-tests/run.sh

```diff
@@ -142,15 +97,21 @@ main() {
 
 	cd $CURRENT_DIR
 
-    # TODO: invoke your tests here
-    cd spring-cloud-kubernetes-core-k8s-client-it
-    ../../mvnw clean install -P it
-    cd ../
-     cd spring-cloud-kubernetes-client-config-it
-    ../../mvnw clean install -P it
-    cd ../
-    cd spring-cloud-kubernetes-configuration-watcher-it
-    ../../mvnw clean install -P it
+	for i in "${PULLING_IMAGES[@]}"; do
+		docker pull $i
```

Comment:
A little logging here about which images we are pulling and loading would be useful I think

Reply:
done